contracts/mocks/ControlledAccessCrowdsaleImpl.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.18;
+
+import "../token/ERC20/ERC20.sol";
+import "../crowdsale/validation/ControlledAccessCrowdsale.sol";
+
+
+contract ControlledAccessCrowdsaleImpl is ControlledAccessCrowdsale {
```

Comment:
seems like we might not need the mock, since it doesn't do anything beyond `ControlledAccessCrowdsale`?

Reply:
I copied the logic from `WhitelistedCrowdsaleImpl.sol`. I believe the idea would be that users would create a contract such as `contract MyTokenCrowdsale is ControlledAccessCrowdsale { ... }` which would specify the parameters fed in the `Crowdsale.sol` constructor.

Reply:
That's what a user would do, for sure, but since this mock doesn't do anything beyond inherit CrontrolledAccessCrowdsale, we can remove it for simplicity. Usually we'd need a mock if we needed to make the constructor payable or something to make testing easier.

Reply:
Perhaps you could combine it with a `MintableCrowdsale` to test the inheritance while we're at it?